examples/tunables_limit_memory.rs

```diff
@@ -131,6 +133,12 @@ impl<T: Tunables> Tunables for LimitingTunables<T> {
     }
 }
 
+impl<T: Tunables> MemoryUsage for LimitingTunables<T> {
+    fn size_of_val(&self, _tracker: &mut dyn MemoryUsageTracker) -> usize {
+        mem::size_of_val(self)
+    }
+}
+
```

Comment:
It's a bit surprising to me that this isn't something the macro can generate. I'm not sure how common of a base case it is, maybe it isn't worth it.

Reply:
It's because of the generic here. I need to dig into `loupe-derive` to see what's going on.

Reply:
Fixed by https://github.com/wasmerio/loupe/pull/14. I'll update this file in #2200 probably.